src/app/Library/CrudPanel/Traits/Create.php

```diff
@@ -73,6 +79,23 @@ public function getRelationFields()
         return $relationFields;
     }
 
+    /**
+     * Associate and dissociate.
+     *
+     * @param  Model
+     * @param  array The form data.
+     * @return Model Model with relationships set up.
+     */
+    public function associateBelongsToRelations($item, array $data)
+    {
+        foreach ($this->getFieldsWithRelationType('BelongsTo') as $relationField) {
+            $item->{$this->getOnlyRelationEntity($relationField)}()
```

Comment:
I find this a little difficult to read. I understand what you went for here - brevity, but... personally I would have made 2 variables for the things used here, so it's easier to read.

No matter, this is just as good as-is, just mentioning 😄 👍

Reply:
Just refactored. I agree with readability over brevity in lots of scenarios. This is one of them. 👍